data/test_domains/default_retrieval_intents.yml

```diff
@@ -22,12 +22,3 @@ responses:
     - text: Bye
   utter_iamabot:
     - text: I am a bot, powered by Rasa.
-
-actions:
-  - utter_chitchat
-  - utter_greet
-  - utter_cheer_up
-  - utter_did_that_help
-  - utter_happy
-  - utter_goodbye
-  - utter_iamabot
```

Comment:
Why were the actions removed in this file?

Reply:
We removed the requirement to list the reponses as actions a long time ago. In copying it over to the `data/test_domains/response_selector_responses_in_domain.yml`, it made sense to test that the `utter_chitchat` action counted as a valid utterance action after the fix even if you didn't list it under `actions`, so I didn't want to copy it over in that case -- to made more sense to remove it from the original one, but i can also put it back, shouldn't affect it either way 😄